src/lightcone.cpp

```diff
@@ -0,0 +1,481 @@
+namespace gevolution{
+
+
+// writes lightcones of particles
+void writeLightcones (
```

Comment:
This function is inconsistent with the current idea of writing lightcone shells.
You should remove this entire file.

Reply:
OK. Solved 513843de3be7ff21b6cb50abc96f118463316d0b